StyleCop.Analyzers/StyleCop.Analyzers.ruleset

```diff
@@ -75,5 +75,6 @@
     <Rule Id="SA1412" Action="Warning" />
     <Rule Id="SA1600" Action="None" />
     <Rule Id="SA1609" Action="Warning" />
+    <Rule Id="SA1629" Action="None" />
```

Comment:
:memo: I wasn't happy with the number of false/questionable positives and ended up disabling the rule.